_layouts/website/_header.html

```diff
@@ -0,0 +1,31 @@
+<header class="header">
+  <div class="header__container">
+    <a href="https://app.n.io" class="nio-logo">
+      <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 702.55 318.26" class="header__logo">
+        <title>nio_logowhite-01</title>
+        <path d="M163.08,313.36v-115c0-33.42-18.27-58.82-53-58.82-33.42,0-56.15,28.07-56.15,61.49V313.36H0V93.25H48.55l3.57,29.85c22.28-21.84,44.56-33,72.19-33,51.69,0,93.13,38.77,93.13,107.84V313.36Z" style="fill: rgb(255, 255, 255);"></path> <path d="M327.08,285.29c0,42.33-64.17,42.33-64.17,0S327.08,243,327.08,285.29Z" style="fill: rgb(255, 255, 255);"></path> <path d="M431.35,31.75c0,42.33-64.17,42.33-64.17,0S431.35-10.59,431.35,31.75Zm-59.26,61V313.36h54.36V92.8Z" style="fill: rgb(255, 255, 255);"></path> <path d="M698.7,203.74c0,63.28-43.22,114.52-114.52,114.52S470.11,267,470.11,203.74c0-62.82,43.67-114.52,113.63-114.52S698.7,140.92,698.7,203.74Zm-174.22,0c0,33.43,20,64.62,59.71,64.62s59.71-31.19,59.71-64.62c0-33-23.17-65.05-59.71-65.05-39.21,0-59.71,32.08-59.71,65.05Z" style="fill: rgb(255, 255, 255);"></path> <path d="M702.55,105.35a5.81,5.81,0,1,1-5.81-5.88,5.57,5.57,0,0,1,5.81,5.88Zm-5.81,5.17a5.16,5.16,0,0,0,0-10.32,5.16,5.16,0,1,0,0,10.32Zm.6-8.33h-2.79v6.17h.76V106h1.34l2,2.32h.9v-.13L697.62,106a1.94,1.94,0,1,0-.29-3.86Zm-2,.74h2a1.22,1.22,0,0,1,.22,2.43h-2.21Z" style="fill: rgb(255, 255, 255);"></path></svg>
+      </a>
+      <input id="header__toggle" type="checkbox" class="header__toggle">
+      <label for="header__toggle" class="header__toggle--label"></label>
+      <nav class="nav">
+        <ul class="nav__list list-unstyled">
+      <li class="nav__list--item">
+        <a id="nav__list--workshop" href="https://nio.school" target="_blank" class="nav__list--anchor">workshop</a>
+      </li>
+      <li class="nav__list--item">
+        <a id="nav__list--docs" href="https://docs.n.io" target="_blank" class="nav__list--anchor">docs</a>
+      </li>
+      <li class="nav__list--item">
+        <a href="https://app.n.io/about" class="nav__list--anchor ">
+            about n.io
```

Comment:
The order we're going with for the nav items is `docs / platform (used to be about n.io but we're renaming to platform) / workshops`